src/components/chat/comment-textbox.tsx

```diff
@@ -50,17 +56,21 @@ export const CommentTextBox: React.FC<IProps> = ({ activeNavTab, numPostedCommen
 
   const handlePostComment = () => {
     // do not send post if text area is empty, only has spaces or new lines
-    const content = commentText.trim().replace(/\s\s+/g, " ");
-    if (!(content === "" || content === "\n" || content === " ")) {
-      onPostComment?.(commentText);
+    const content = trimContent(commentText);
+    const trimmedCommentText = content[0] as string;
+    const isEmpty = content[1];
```

Comment:
You can replace this with:
```suggestion
    const [trimmedText, isEmpty] = trimContent(commentText);
```